less/bin-basket.less

```diff
@@ -0,0 +1,187 @@
+@bin-basket-circle-size: 150px;
+bin-basket {
+  cursor: pointer;
+  &.modern-basket {
```

Comment:
Seems like a strange name, wouldn't it be better to indicate this is about the dropdown-link variant of the basket? Or is it more general?

Reply:
This is to indicate that it is the basket that is being modernized, we don't want these changes to apply on all the basket links.